app/src/main/java/com/casadelfuego/omnikiller/ui/main/MainFragment.kt

```diff
@@ -25,6 +23,8 @@ class MainFragment: Fragment() {
     super.onActivityCreated(savedInstanceState)
     viewModel = ViewModelProviders.of(this).get(MainViewModel::class.java)
     // TODO: Use the ViewModel
+
+    addCardButton.setOnClickListener(Navigation.createNavigateOnClickListener(R.id.action_mainFragment_to_addCardFragment, null))
```

Comment:
Encurta

Reply:
Quebra, quequebra, quequebra, quebra

Reply:
O que tu quer dizer com encurta? 
se for `addCardButton.onClickListener  = blalblabla`, não funciona pq clickListener não é property